packages/insured-bridge-relayer/src/RelayerConfig.ts

```diff
@@ -13,6 +13,12 @@ export const bridgeDepositBoxDeployData = {
   288: { blockNumber: 223808 },
 };
 
+// Following settings can be overridden to optimize L1 client event search for select events. For example,
+// the "DepositRelayed" event search request could return > 10,000 return values so we need to shorten the block
+// search using these parameters because some node providers like Infura limit event search return values to 10,000.
+export const bridgePoolEarliestBlockToSearch = 13545377;
+export const bridgePoolMaxBlocksToSeach = 1_000_000;
```

Comment:
Roughly how many of these windows do you think we'll need to search through to get through the whole range?

Reply:
2, there are 1.1mil blocks since deployment block